### PR TITLE
Remove reference doc from READMEs

### PR DESCRIPTION
**Pull request by mazhelez**
#### Summary 
Remove reference doc from READMEs. Reference doc is available at [https://microsoft.github.io/BCApps/](https://microsoft.github.io/BCApps/).
#### Work Item(s) 
Fixes [AB#501035](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/501035)

### CODE REVIEW

File: src/System Application/App/Advanced Settings/README.md [hunk 1]
```diff
@@ -1,28 +1,3 @@
 The modules exposes advanced settings page and related integration events.
  
 
-# Public Objects
```
Comment (aholstrup1) [inline src/System Application/App/Advanced Settings/README.md:4]: Do we have a way of populating the reference docs on https://microsoft.github.io/BCApps/ with documentation from the release branches?

Comment (mazhelez) [reply src/System Application/App/Advanced Settings/README.md:4]: No, but wasn't it agreed that reference documentation on release branches will live on _learn_? E.g.: https://learn.microsoft.com/en-us/dynamics365/business-central/application/system-application/module/system-application

Not sure how this is versioned though. In any case, the doc is also in code for the release branches.

File: src/System Application/App/Advanced Settings/README.md [hunk 2]
```diff
@@ -1,28 +1,3 @@
 The modules exposes advanced settings page and related integration events.
```
Comment (JesperSchulz) [inline src/System Application/App/Advanced Settings/README.md:3]: Any chance we can link to the reference docs for now? Just to kick things off and make people aware where the documentation has moved to?

Comment (mazhelez) [reply src/System Application/App/Advanced Settings/README.md:3]: Sounds like a good idea! Will do.

**Review by blrobl — APPROVED** [review]

**Review by aholstrup1 — APPROVED** [review]

**Pull request merged**